UnityProject/Assets/Scripts/Medical/DNAscanner.cs

```diff
@@ -8,9 +8,7 @@ public class DNAscanner : ClosetControl, IAPCPowered
 	public LivingHealthBehaviour occupant;
 	public string statusString;
 	[SyncVar(hook = nameof(SyncPowered))] public bool powered;
-	public Sprite closedWithOccupant;
-	public Sprite doorClosedPowerless;
-	public Sprite doorOpenPowerless;
+	public SpriteHandler spriteHandler;
```

Comment:
this can be private, and get it using GetComponent<SpriteHandler> during Awake()